tests/handlers/test_e2e_keys.py

```diff
@@ -171,6 +171,47 @@ def test_claim_one_time_key(self):
             },
         )
 
+    @defer.inlineCallbacks
+    def test_fallback_key(self):
```

Comment:
It would be nice to test some of the error scenarios also -- e.g. the fallback keys not being the correct format or invalid types.

Reply:
The server side generally doesn't care about the key formats or types (this is the same with normal one-time keys).

Reply:
Should there be a test for there being both a "normal" key and a fallback key?